source/install/install-ubuntu-2004-mysql.rst

```diff
@@ -0,0 +1,55 @@
+.. _install-ubuntu-2004-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL.
+
+**To install MySQL on Ubuntu Server 20.04:**
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Install MySQL.
+
+  ``sudo apt install mysql-server``
+
+3. Run ``mysql_secure_installation`` and follow the instructions.
+
+  ``sudo mysql_secure_installation``
+
+4. Log in to MySQL as root.
+
+  ``sudo mysql``
+
+5. Create the Mattermost user 'mmuser'.
```

Comment:
```suggestion
5. Create the Mattermost user *mmuser*.
```

Reply:
Note: This is a change that needs to be made across all the install guides.

Reply:
> Note: This is a change that needs to be made across all the install guides

Noted 👍